### ISSUE TITLE
Create accessible color combination for titles

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10071]
## Description
In the CMS on the view screen, there are read only blocks of text. Some blocks have titles and these titles (Hex 757576) do not pass WCAG 2.0 AA color contrast requirements against the light gray (HEX f1f1f1) background of the surrounding block or the red background (Hex f9dede) if the content is set to draft 

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/182943730-2add24b0-1124-4d71-8a19-4ebbd3bbbcc4.png)

![image](https://user-images.githubusercontent.com/106678594/194409745-6ac086af-195b-4560-a3ff-0e99df1ea27c.png)

## Accessibility Standard

WCAG version 2.0 AA, [Criterion 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

## Acceptance Criteria
- [x] Confirm that updating title text color is the best option - rather updating the background color
- [x] Confirm read only block is correctly documented in the Design system - including the title text and background colors
- [ ] Technical review for updating title text color in CMS to Gray Darker (Hex 494440)
- [ ] Change management is consulted
- [x] Implementation ticket is created

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by BlakeOrgan** [comment]
These should be updated to the base color `#212121` to ensure that they are ADA compliant against all background colors that we use.

I'm starting to see patterns emerge where we use a light grey text in places where the text would be better off as our base grey. I assume this was originally done to provide visual hierarchy, but this would be better done through adjusting the font size or weight rather than the color in my opinion.

**Comment by BlakeOrgan** [comment]
Here's the ticket I created to implement these changes. https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11455
